src/Core/src/Dispatching/Dispatcher.Android.cs

```diff
@@ -64,7 +69,32 @@ public void Start()
 
 			IsRunning = true;
 
-			_handler.PostDelayed(OnTimerTick, (long)Interval.TotalMilliseconds);
+			Post();
+		}
+
+		void Post()
+		{
+			if (IsCallbackPosted())
+			{
+				return;
+			}
+
+			_handler.PostDelayed(_runnable, (long)Interval.TotalMilliseconds);
+
+			if (!OperatingSystem.IsAndroidVersionAtLeast(29))
```

Comment:
what changed in API level 29?

Reply:
The `hasCallbacks()` method did not exist until API 29.
https://developer.android.com/reference/android/os/Handler#hasCallbacks(java.lang.Runnable)